src/components/auth/SignInScreen.js

```diff
@@ -66,20 +70,17 @@ export default class SignInScreen extends React.Component {
   render() {
     return (
       <SafeAreaView style={styles.container}>
-        <KeyboardAvoidingView style={styles.container} behavior='padding' enabled>
           <TouchableWithoutFeedback
```

Comment:
## Test if Keyboard dismisses when active.

### If the test fails,  nest KeyboardAvoidingView in TouchableWithoutFeedback, in other terms swap the tags.

Reply:
the keyboard does in fact dismiss correctly but there is a difference between our screens (home and sign in). however your menubar in home is nested inside the keyboardavoidingview. i'm not sure if this has been discussed yet but personally i like the menubar to stay put when the rest of the content moves out of the way. it leaves you with access to all the buttons and the logo at the bottom is the only thing hidden. pressing on this menubar will not dismiss the keyboard like pressing on the sandy area will. should we make it the same and if so which one?

Reply:
Keep whichever doesn't have the menubar moving around. it should stay static